src/components/Navigation.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Navigation = () => {
```

Comment:
As mentioned above, I truly love that you separated navigation into a new component 👏